Samples/WebSignIn/Common (iOS)/Keychain/Keychain+Token.swift

```diff
@@ -22,17 +22,23 @@ public extension Keychain {
     private static let oktaMobileSdkAccessGroup = "com.okta.mobile-sdk.shared"
     
     static func saveDeviceSSO(_ token: Token) throws {
-        try [Token.Kind.idToken, Token.Kind.deviceSecret].forEach { kind in
-            try token.token(of: kind)
-                .flatMap { $0.data(using: .utf8) }
-                .map {
-                    Keychain.Item(account: "Okta-\(kind.rawValue)",
-                                  service: "Okta",
-                                  accessGroup: oktaMobileSdkAccessGroup,
-                                  value: $0)
-                }
-                .map{ try $0.save() }
+        try [Token.Kind.idToken, Token.Kind.deviceSecret]
+            .forEach { try saveDeviceSSO(token, kind: $0) }
+    }
+    
+    static func saveDeviceSSO(_ token: Token, kind: Token.Kind) throws {
+        guard let tokenString = token.token(of: kind),
+              let data = tokenString.data(using: .utf8)
+        else {
+            return
         }
+        
+        try Keychain
+            .Item(account: "Okta-\(kind.rawValue)",
+                  service: "Okta",
```

Comment:
Do these need to be namespaced to Okta? May be worthwhile to make this configurable by the consumer, as it'll delegate global keychain management/migration to the consumer.

Reply:
This is just for the purposes of the sample app. A real app wouldn't do things in this fashion.